src/styles/components/_errorBox.scss

```diff
@@ -0,0 +1,40 @@
+/*------------------------------------*\
+  # Error Box
+\*------------------------------------*/
+
+.error-box-container {
+    left: 50%;
```

Comment:
Indent here, you're using two-space tab indent. Check the rest of the scss files